src/worldstate/include/routines/State.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Created By:  Joel Ahn
+ * Created On:  May 6th, 2018
+ * Description: Superclass detailing the individual nodes
+ *              of the world state finite state machine
+ */
+
+#ifndef PROJECT_ROUTINE_H
+#define PROJECT_ROUTINE_H
+
+#include <ros/ros.h>
+#include <worldstate/StateMsg.h>
+
+/*
+ * Make sure to provide for each State subclass the possible
+ * StateMsg types that it can publish i.e. to which other States
+ * that the World Finite State Machine can transition from
+ * the current one to. Describe before the class declaration as such:
+ *
+/*** Communicating States {States that it can possibly transition to} ***/
```

Comment:
Similar comment to above, not sure what this is saying, should remove or modify.